### DISCUSSION TITLE
buffer for result of evaluation?

### DISCUSSION THREAD

**Discussion by justin2004** [liquidz/vim-iced#406] [General]
i know there is a vim buffer for stdout.
e.g. (print "something")

but is it possible to have a vim buffer for the values that result from evaluation (that don't go to stdout)?
e.g.
"something"
(+ 1 2)

**Comment by liquidz** [comment]
@justin2004 
You can do it with hook feature.
https://liquidz.github.io/vim-iced/#hook

e.g.
```vim
call iced#hook#add('evaluated', {
     \ 'type': 'function',
     \ 'exec': {d -> iced#buffer#stdout#append(d['result']['value'])},
     \ })
```

**Reply by justin2004** [reply]
this worked. thanks!
